server/routerlicious/pnpm-lock.yaml

```diff
@@ -3177,6 +3177,11 @@ packages:
 
   /@fluidframework/protocol-definitions/3.1.0:
```

Comment:
Looks like we still have the old version in the package lock. Maybe something didn't update? Would be good to see whats keeping that installed.

Reply:
yeah it seems like the previous version of packages:  /@fluidframework/server-services-core/2.0.0:
For the current version of packages, they are updated. Current is 3.0